app/features/quizzify/tools.py

```diff
@@ -83,16 +318,37 @@ def load(self) -> List[Document]:
 
                     doc = Document(page_content=page_content, metadata=metadata)
                     documents.append(doc)
+
+            
+            elif file_type in ["doc", "docx"]:
+                page_content = extract_text_from_docx(file)
+                metadata = {"source": file_type}
+                doc = Document(page_content=page_content, metadata=metadata)
+                documents.append(doc)
+                
+
+            elif file_type in ["ppt", "pptx"]:
+                page_content = extract_text_from_pptx(file)
+                metadata = {"source": file_type}
+                doc = Document(page_content=page_content, metadata=metadata)
+                documents.append(doc)
+
+            elif file_type == "csv":
+                df = pd.read_csv(file)
+                page_content = df.to_string()
+                metadata = {"source": file_type}
+                doc = Document(page_content=page_content, metadata=metadata)
+                documents.append(doc)
                     
             else:
                 raise ValueError(f"Unsupported file type: {file_type}")
             
         return documents
 
 class LocalFileLoader:
```

Comment:
Perhaps we should deprecate the `LocalFileLoader` class altogether since we are receiving the files as a URL from the frontend